build/setup-operator-sdk.sh

```diff
@@ -9,8 +9,11 @@ if ! [ -e "./operator-sdk" ]; then
     *)          file=operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu;;
   esac
 
+ARCH=$(uname -m)
```

Comment:
Could you please apply the same change to build/setup-operator-sdk-e2e-tests.sh?

We use a different version of the operator-sdk for the end-to-end tests but we need to make sure this version also uses the proper arch.

Reply:
Yes @jmesnil . I have made the required changes.